docs/codeql/codeql-overview/codeql-changelog/codeql-cli-2.21.1.rst

```diff
@@ -37,14 +37,6 @@ Bug Fixes
 Query Packs
 -----------
 
-New Features
```

Comment:
The changes in this file are fine - I see it is just moving around the GitHub Actions GA notes. That's because we wrote them manually after the fact, and the changelog generator is using a different order for the various sections.